aiida_firecrest/remote_path.py

```diff
@@ -1,617 +0,0 @@
-"""A pathlib.Path-like object for accessing the file system via the Firecrest API.
```

Comment:
Why is this file removed?

Reply:
This file originally written by @chrisjsewell, [got merged](https://github.com/eth-cscs/pyfirecrest/pull/43) into `pyfirecrest`, so we now remove from here and import from there.